.env

```diff
@@ -0,0 +1,46 @@
+APP_NAME="KA-Huis"
```

Comment:
@cyrildewit  you shouldn't commit .env file. This is due to security reasons a bad thing to do.